build/common.sh

```diff
@@ -330,7 +330,7 @@ function kube::build::has_docker() {
 }
 
 function kube::build::has_ip() {
-  ip -Version | grep 'iproute2' &> /dev/null
```

Comment:
This line could be changed to:
```
ip -Version 2> /dev/null | grep 'iproute2' &> /dev/null
```
This would suppress the error message:
```
/opt/k8sgopath/src/k8s.io/kubernetes/build/../build/common.sh: line 333: ip: command not found
```

This is the expected result of this function if the system lacks the `ip` utility. It will exit with a non-zero error code, which results in the script falling back to using `ifconfig`.